### ISSUE TITLE
Integrated Progress Bar and Improved Default Styling

### ISSUE THREAD

**Issue by dan-compton** [odysseyscience/react-s3-uploader#19]
Do you have any thoughts on improving the component stylistically and/or including a react-bootstrap progress bar?  This might be outside of the project scope, though many other (similar) components like react-dropzone come prepackaged with styles.

I have an extremely rough version of what I'm talking about at assignmentexchangestaging.herokuapp.com  test_tutor1:test_tutor and click the paperclip icon OR see image below.

<img width="769" alt="screen shot 2015-07-25 at 9 58 59 am" src="https://cloud.githubusercontent.com/assets/12177003/8889937/d385bfe2-32b3-11e5-9b96-80e08799a241.png">

**Comment by seanadkinson** [comment]
Yeah, we have our own styling on the progress bar, which looks real nice. With this repo, I didn't want to worry about browser support etc, but I think that people would expect and appreciate a little more done for them. 

Maybe you can send me your component and styles, and I'll grab ours as well, and I can put together some wrapper components that style it better, while still providing the raw component for people to customize.

**Comment by dan-compton** [comment]
Sounds good.  I'll revise mine and get back to you on this thread.  I respect your attitude on project scope.  I'm relatively new to react (more of a backend guy) and I would love to see your stuff as well.

**Comment by kliu4** [comment]
Any new updates?
